ci/github/unit-testing/settings-library.bash

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit   # abort on nonzero exitstatus
+set -o nounset   # abort on unbound variable
+set -o pipefail  # don't hide errors within pipes
+IFS=$'\n\t'
+
+install() {
+    bash ci/helpers/ensure_python_pip.bash
+    pushd packages/settings-library; pip3 install -r requirements/ci.txt; popd;
+    pip list -v
+}
+
+test() {
+    pytest --cov=settings_library --durations=10 --cov-append \
+          --color=yes --cov-report=term-missing --cov-report=xml --cov-config=.coveragerc \
+          -v packages/settings-library/tests
```

Comment:
are these tests real unit tests? if yes you can add the the pytest-xdist to use multiple CPUs

Reply:
yes. pytest-xdist looks definitively useful but here tests take 6 secs. I rather do that in other larger test suite